src/mol-gl/shader/shadows.frag.ts

```diff
@@ -80,20 +80,19 @@ float screenSpaceShadow(const in vec3 position, const in vec3 lightDirection, co
         rayCoords = uProjection * vec4(rayPos, 1.0);
         rayCoords.xyz = (rayCoords.xyz / rayCoords.w) * 0.5 + 0.5;
 
-        if (outsideBounds(rayCoords.xy))
-            return 1.0;
+        //if (outsideBounds(rayCoords.xy))
+        //    return 1.0;
 
         // Compute the difference between the ray's and the camera's depth
         float depth = getDepth(rayCoords.xy);
-        float viewZ = getViewZ(depth);
-        float zDelta = rayPos.z - viewZ;
+        float sampleViewZ = screenSpaceToViewSpace(vec3(rayCoords.xy, depth), uInvProjection).z;
```

Comment:
getViewZ should give the same result (but calculated cheaper). I did not see any visual difference, do you have an example?